op-service/txmgr/txmgr.go

```diff
@@ -745,13 +833,18 @@ func (m *SimpleTxManager) suggestGasPriceCaps(ctx context.Context) (*big.Int, *b
 	m.metr.RecordTipCap(tip)
 
 	// Enforce minimum base fee and tip cap
-	if minTipCap := m.cfg.MinTipCap; minTipCap != nil && tip.Cmp(minTipCap) == -1 {
-		m.l.Debug("Enforcing min tip cap", "minTipCap", m.cfg.MinTipCap, "origTipCap", tip)
-		tip = new(big.Int).Set(m.cfg.MinTipCap)
+	m.cfgLock.Lock()
+	minTipCap := m.cfg.MinTipCap
+	minBaseFee := m.cfg.MinBaseFee
+	m.cfgLock.Unlock()
```

Comment:
Could use the getter (although locking twice, unless using atomics instead, see other comment).